src/Uno.UI/UI/Xaml/IFrameworkElement.cs

```diff
@@ -152,10 +157,12 @@ public static void Initialize(IFrameworkElement e)
 #endif
 			}
 
-#if XAMARIN_ANDROID
-			if (Debugger.IsAttached)
+#if __ANDROID__
+			if (e is View view)
 			{
-				(e as View).ContentDescription = e.GetType().ToString();
+				// TODO: This causes Android to cycle through every visible IFrameworkElement every time the accessibility focus is changed
+				// and calls the overriden InitializeAccessibilityNodeInfo method. This could eventually cause performance issues when accessibility is enabled.
```

Comment:
When is 'eventually'? Is this a serious performance concern?

Reply:
'eventually' is if we have complex UI with lots of elements. I've only noticed lags when cycling through accessible elements in debug.

The performance problem seem to be due to interop. It should only affect focus change on devices with TalkBack turned on. A possible optimization could be to set this value to false if an element's `CreateAutomationPeer` returns `null`.